model/project_ref.go

```diff
@@ -2246,36 +2246,36 @@ func (p *ProjectRef) GetGithubProjectConflicts() (GithubProjectConflicts, error)
 
 // ValidateProjectCreation returns an error if the total or owner/repo project limit set by the admin settings has been reached.
```

Comment:
This comment isn't quite right since it returns an error also if there's an error, and the boolean determines if its an error or warning, but we intend to tweak this to handle the front end right (to use status codes instead)? In which case we can just handle it all later.